docs/index.md

```diff
@@ -472,7 +472,10 @@ Knowing which companies are using this library is important to help prioritize t
 [**Vertex Pharmaceuticals**](https://www.vrtx.com/){target="_blank" rel="nofollow"}
 { .card }
 
-[**Alma Media**](https://www.almamedia.fi/en/){target="_blank" rel="nofollow}
+[**Alma Media**](https://www.almamedia.fi/en/){target="_blank" rel="nofollow"}
+{ .card }
+
+[**Banxware**](https://www.banxware.com){target="_blank" rel="nofollow"}
```

Comment:
We try to keep the alphabetical order. Can you please change this?

Reply:
Alma Media was already last - do you want me to put it at the top?